services/user.py

```diff
@@ -0,0 +1,43 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
```

Comment:
According to the checklist, you should use `get_user_model()` instead of directly importing `User` from `db.models`. This is a best practice in Django to avoid potential issues with custom user models.